### PR TITLE
Replacing lodash map with native map method

### PR DESCRIPTION
**Pull request by pnkapadia64**
#302 

Replacing lodash `_.map` usages with native `.map`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: __tests__/buildFiles.test.js [hunk 1]
```diff
@@ -93,13 +93,13 @@ describe('buildFiles', () => {
 
   it('should throw if build path doesn\'t have a trailing slash', () => {
     expect(
-      buildFiles.bind(null, dictionary, platformWithBadBuildPath),
+      buildFiles.bind(null, dictionary, platformWithBadBuildPath)
```
[the remark this answers is not in the document]
Comment (pnkapadia64) [reply __tests__/buildFiles.test.js]: The pre-commit hook (lint script) fails without this. Is it fine if I ignore that?

File: __tests__/buildFiles.test.js [hunk 2]
```diff
@@ -93,13 +93,13 @@ describe('buildFiles', () => {
...
     ).toThrow('Build path must end in a trailing slash or you will get weird file names.');
   });
 
   it('should throw if missing a format', () => {
     expect(
-      buildFiles.bind(null, dictionary, platformWithoutFormatter),
+      buildFiles.bind(null, dictionary, platformWithoutFormatter)
```
[the remark this answers is not in the document]
Comment (dbanksdesign) [reply __tests__/buildFiles.test.js]: That is weird, it shouldn't. Can you be sure you are using our version of eslint and not a globally installed version? Double check you have eslint in node_modules/.bin and make sure you run npm ci to install exact dependency versions. You also might have an issue with your $PATH, you can run npm run lint --verbose to make sure local executables are before any global paths.

**Comment by pnkapadia64** [comment]
Hey @dbanksdesign @chazzmoney, I've removed the unnecessary changes in test files that came up before as lint errors. Thank you.

**Comment by chazzmoney** [comment]
:shipit:

**Pull request merged**